### PR TITLE
Jfb 2 stadium sector function

### PR DESCRIPTION
**Pull request by Nedzeya**

### CODE REVIEW

File: src/sample/complete/Ticket.java
Comment (Alexmansar) [inline src/sample/complete/Ticket.java]: No comment needed ))

File: src/sample/complete/TicketService.java
Comment (Alexmansar) [inline src/sample/complete/TicketService.java]: the var called **ticketByStadiumSector**, the method getTicket**_s_**ByStadiumSector
jsut use the same style please))

File: src/sample/complete/TicketService.java
Comment (Alexmansar) [inline src/sample/complete/TicketService.java]: Generally, using generic is a good practice, but in this case, I think we can't reuse it) because your call to System.out.println() is only related to the ticket))))

and we need to update the name. The name means we expect the result in String))))) 
Use anything starting with print... printTickets f.e

File: src/sample/complete/TicketService.java
Comment (Alexmansar) [inline src/sample/complete/TicketService.java]: In this case you get the id twice)) I think we can use List instead of Map from line 33

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@chingiz-mazhitov @Nedzeya Hi, please check comments)

File: src/sample/complete/Ticket.java
Comment (Nedzeya) [reply src/sample/complete/Ticket.java]: XD

File: src/sample/complete/TicketService.java
Comment (Nedzeya) [reply src/sample/complete/TicketService.java]: (I think we can use List instead of Map from line 33 )- Do you mean to chenge return type for  private static Map<Integer, Ticket> getTicketsByStadiumSector(char sector) to List?

Comment (Alexmansar) [reply src/sample/complete/TicketService.java]: Yes, correct)

Comment (chingiz-mazhitov) [reply src/sample/complete/TicketService.java]: > (I think we can use List instead of Map from line 33 )- Do you mean to chenge return type for private static Map<Integer, Ticket> getTicketsByStadiumSector(char sector) to List?

Basically, I'm ready to merge it, if you're not gonna change here anything I guess...

Comment (Nedzeya) [reply src/sample/complete/TicketService.java]: this is my PR and I think that I should merge it... Isn't it?

Comment (Nedzeya) [reply src/sample/complete/TicketService.java]: I've pushed my changes. Could you please take a look?

**Review by Alexmansar — APPROVED** [review]
@Nedzeya @chingiz-mazhitov please, merge it)))

**Pull request merged**
